nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-pubsub-processors/src/main/java/org/apache/nifi/processors/kafka/pubsub/AbstractKafkaProcessor.java

```diff
@@ -141,6 +166,10 @@
         SHARED_DESCRIPTORS.add(CLIENT_ID);
         SHARED_DESCRIPTORS.add(SECURITY_PROTOCOL);
         SHARED_DESCRIPTORS.add(KERBEROS_PRINCIPLE);
+        SHARED_DESCRIPTORS.add(SSL_KEY_PASSWORD);
```

Comment:
This feels very odd to me to provide properties for the passwords but not the keystore and truststore. I do realize that the keystore and truststore can be specified via user-defined properties, but it adds to the complexity of configuring this, as the user has to lookup the appropriate property names. This is a usability miss.

Why do we not simply expose a property for an SSLContextService? We could then pull the appropriate values from the SSLContextService via getKeyStorePassword(), getKeyStoreFile(), etc.